connectivity/builder/builder.go

```diff
@@ -79,23 +79,86 @@ type testBuilder interface {
 	build(ct *check.ConnectivityTest, templates map[string]string)
 }
 
-// NetworkPerformanceTests builds the network performance connectivity tests.
-func NetworkPerformanceTests(ct *check.ConnectivityTest) error {
-	return getTests(ct, []testBuilder{networkPerf{}})
+// GetTestSuites returns a slice of connectivity test builder functions.
+// Each function contains set of tests according to the provided parameters.
+func GetTestSuites(params check.Parameters) ([]func(connTests []*check.ConnectivityTest, extra func(ct *check.ConnectivityTest) error) error, error) {
+	switch {
+	case params.Perf:
+		return []func(connTests []*check.ConnectivityTest, _ func(ct *check.ConnectivityTest) error) error{
```

Comment:
Giving `func(connTests []*check.ConnectivityTest, _ func(ct *check.ConnectivityTest) error) error` a type name would make this function more readable.

Reply:
Fixed.